components/slider/Slider.razor.cs

```diff
@@ -416,9 +426,39 @@ private async void OnMouseDown(MouseEventArgs args)
             _mouseDown = !Disabled;
         }
 
+        private MouseEventArgs _edgeClickArgs;
+        private bool? _edgeClicked;
+        private void OnMouseDownEdge(MouseEventArgs args, bool right)
+        {
+            _right = right;
+            _edgeClickArgs = args;
+        }
+
+        private bool IsMoveInEdgeBoundary(JsonElement jsonElement)
+        {
+            if (_edgeClicked == null)
+            {
+                double clientX = jsonElement.GetProperty("clientX").GetDouble();
+                double clientY = jsonElement.GetProperty("clientY").GetDouble();
+                bool altKey = jsonElement.GetProperty("altKey").GetBoolean();
+                bool ctrlKey = jsonElement.GetProperty("ctrlKey").GetBoolean();
+                bool metaKey = jsonElement.GetProperty("metaKey").GetBoolean();
+                bool shiftKey = jsonElement.GetProperty("shiftKey").GetBoolean();
+
+                _edgeClicked = _edgeClickArgs.ClientX == clientX
+                            && _edgeClickArgs.ClientY == clientY
+                            && _edgeClickArgs.CtrlKey == ctrlKey
+                            && _edgeClickArgs.MetaKey == metaKey
+                            && _edgeClickArgs.AltKey == altKey
+                            && _edgeClickArgs.ShiftKey == shiftKey;
```

Comment:
The `_edgeClickArgs` would be null.

```suggestion
                _edgeClicked = _edgeClickArgs != null
                            && _edgeClickArgs.ClientX == clientX
                            && _edgeClickArgs.ClientY == clientY
                            && _edgeClickArgs.CtrlKey == ctrlKey
                            && _edgeClickArgs.MetaKey == metaKey
                            && _edgeClickArgs.AltKey == altKey
                            && _edgeClickArgs.ShiftKey == shiftKey;
```

Reply:
Yes, exactly. Among other things, I have this fixed in coming commit.

Reply:
Just tried to commit but I am getting this error
```
husky > pre-commit (node v14.15.1)
/d/dotnet/ant-design-blazor_forked/node_modules/.bin/lint-staged: line 5: cygpath: command not found
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'D:\lint-staged\bin\lint-staged.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint  as runMain 
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
husky > pre-commit hook failed (add --no-verify to bypass) 
```
Of course I can run suggested --no-verify but should I do it? What is this? I never used husky. I tries rebuilding, restarting, running ```npm install```...

Reply:
It is used to check the if the commit message is following the commit message standard.
You can skip it because I would change the commit message before merge PRs.